ui/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "danaswap-ui",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "node build.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
shouldnt these be `dependencies`

Reply:
these deps are here for building the static HTML/CSS, they will not be used in production